recipes-tag/mpi4py/meta.yaml

```diff
@@ -0,0 +1,54 @@
+# Copied and pasted by Leo Fang from the following commit:
+# https://github.com/conda-forge/mpi4py-feedstock/blob/ff2cc11800303988538b8ce3e9707ea97d5bdd2e/recipe/meta.yaml
+
+{% set build = 1004 %}
+{% set version = '3.0.0' %}
+{% set sha256 = 'b457b02d85bdd9a4775a097fac5234a20397b43e073f14d9e29b6cd78c68efd7' %}
+
+package:
+  name: mpi4py
+  version: {{ version }}
+
+source:
+  fn: mpi4py-{{ version }}.tar.gz
+  url: https://bitbucket.org/mpi4py/mpi4py/downloads/mpi4py-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  script: pip install --no-deps .
+  # TODO: build with msmpi when it works
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }} # [mpi == 'openmpi']
```

Comment:
The condition test is the only deviation from conda-forge for avoiding a gcc error for "unrecognized command line option -fstack-protector-strong and -fno-plt" for MPICH. Don't know what's wrong about it.

Reply:
Requiring `conda-build >=3.17.0` also resolves this issue for some reason, and the condition test is no longer needed.